conda-forge.yml

```diff
@@ -12,3 +12,4 @@ provider:
 test_on_native_only: true
 conda_build:
   pkg_format: '2'
+remote-ci-setup: ["conda-forge-ci-setup=3", "libsolv<0.7.23"]  # https://github.com/conda/conda-build/issues/4750
```

Comment:
```suggestion
remote_ci_setup: ["conda-forge-ci-setup=3", "libsolv<0.7.23"]  # https://github.com/conda/conda-build/issues/4750
```

Reply:
[See this comment for context](https://github.com/conda-forge/conda-build-feedstock/pull/197/files#r1150397721).

This might need a rerender (use the bot command) too.